### ISSUE TITLE
raft print log flood about snapshot when one follower die

### ISSUE THREAD

**Issue by yichengq** [coreos/etcd#3292]
```
2015-08-13 15:21:39.347788 I | raft: 6e3bd23ae5f1eae0 snapshot failed, resumed sending replication messages to 924e2e83e93f2560 [next = 1, match = 0, state = ProgressStateProbe, waiting = false, pendingSnapshot = 0]
2015-08-13 15:21:39.447394 I | raft: 6e3bd23ae5f1eae0 [firstindex: 1185606, commit: 1194978] sent snapshot[index: 1190605, term: 4] to 924e2e83e93f2560 [next = 1, match = 0, state = ProgressStateProbe, waiting = false, pendingSnapshot = 0]
2015-08-13 15:21:39.447413 I | raft: 6e3bd23ae5f1eae0 paused sending replication messages to 924e2e83e93f2560 [next = 1, match = 0, state = ProgressStateSnapshot, waiting = true, pendingSnapshot = 1190605]
2015-08-13 15:21:39.449271 I | raft: 6e3bd23ae5f1eae0 snapshot failed, resumed sending replication messages to 924e2e83e93f2560 [next = 1, match = 0, state = ProgressStateProbe, waiting = false, pendingSnapshot = 0]
2015-08-13 15:21:39.546598 I | raft: 6e3bd23ae5f1eae0 [firstindex: 1185606, commit: 1195757] sent snapshot[index: 1190605, term: 4] to 924e2e83e93f2560 [next = 1, match = 0, state = ProgressStateProbe, waiting = false, pendingSnapshot = 0]
2015-08-13 15:21:39.546616 I | raft: 6e3bd23ae5f1eae0 paused sending replication messages to 924e2e83e93f2560 [next = 1, match = 0, state = ProgressStateSnapshot, waiting = true, pendingSnapshot = 1190605]
2015-08-13 15:21:39.547850 I | raft: 6e3bd23ae5f1eae0 snapshot failed, resumed sending replication messages to 924e2e83e93f2560 [next = 1, match = 0, state = ProgressStateProbe, waiting = false, pendingSnapshot = 0]
```

It could print 30 lines per second, which happens when leader keep sending snapshot to dead follower. We should make log much readable.

**Comment by xiang90** [comment]
@yichengq We need to investigate why we keep on sending snapshot to a dead member at such a high frequency.

**Comment by yichengq** [comment]
Current policy is when snapshot sent fails, it waits one heartbeat before enabling sending snapshot again. So it sends snapshot once per 100ms.

This needs to be log backoff maybe.

**Comment by xiang90** [comment]
Fixed by #3324
